Permanent/ViewControllers/AccountInfoViewController.swift

```diff
@@ -144,33 +144,9 @@ extension AccountInfoViewController: UITextFieldDelegate {
     }
     
     private func switchBasedNextTextField(_ textField: UITextField) {
```

Comment:
can we rename this method to something more meaningful? or maybe just get rid of it and move the code in textFieldShouldReturn?

Reply:
Sure, we can remove "switchBasedNextTextField" method.